### ISSUE TITLE
Error in vignette execution

### ISSUE THREAD

**Issue by seandavi** [hubentu/BioC19CWL#13]
```
> library(RcwlPipelines)
Loading required package: BiocFileCache
Loading required package: dbplyr
> input1 <- InputParam(id = "sth")
> echo <- cwlParam(baseCommand = "echo", inputs = InputParamList(input1))
> echo$sth <- "Hello World!"
> echo
class: cwlParam
 cwlClass: CommandLineTool
 cwlVersion: v1.0
 baseCommand: echo
inputs:
  sth (string):  Hello World!
outputs:
output:
  type: stdout
r1 <- runCWL(echo, outdir = tempdir())
r1

> r1 <- runCWL(echo, outdir = tempdir())
INFO Final process status is success
Error in idx[1]:(idx[2] - 1) : NA/NaN argument
```

This looks like a bug? If you could take a quick look, that would be great, as we are trying to finalize the AMI. Thanks!

**Comment by hubentu** [comment]
Likes like it run successfully, but couldn't print out logs. Could you give me a temporary access to the AMI or the session info? Thanks!

**Comment by hubentu** [comment]
Hi @seandavi , I have submitted a fix for the error (Rcwl 1.1.5 in devel). The latest version of cwltool put a new "INFO" in front of output logs. Please have a test. Thanks!
